lib/puppet/face/certregen.rb

```diff
@@ -168,7 +168,7 @@
       rv = {succeeded: [], failed: []}
       PuppetX::Certregen::CA.certnames.each do |certname|
         begin
-          PuppetX::Certregen::CA.distribute_cacert(certname, config)
+          PuppetX::Certregen::CA.distribute(certname, config)
```

Comment:
Probably would be good to update the `description` for the `redistribute` action with some verbiage indicating that both the CA and CRL would be distributed.